Rakefile

```diff
@@ -165,6 +165,16 @@ namespace :release do
          "package: update version info to #{version} (#{new_release_date})")
       sh("git", "push")
     end
+
+    desc "Validate version for release"
+    task :validate do
+      if version != BASE_VERSION
+        message = "You must NOT specify VERSION= for 'rake release'. "
+        message += "You must run 'rake dev:version:bump NEW_VERSION=#{version}' "
+        message += "before 'rake release'."
+        raise message
+      end
+    end
```

Comment:
Could you move this before `task :update`?

Reply:
fix: [8b0ad9f](https://github.com/groonga/groonga/pull/2504/commits/8b0ad9f10d4eb4094fa14c2b04e9c7a3f1596aaa) Sure.